.travis.yml

```diff
@@ -189,7 +189,7 @@ jobs:
         - travis_retry ./scripts/pod_lib_lint.rb FirebaseInAppMessagingDisplay.podspec --use-libraries
         - travis_retry ./scripts/pod_lib_lint.rb FirebaseInAppMessagingDisplay.podspec --use-modular-headers
 
-    # Split from previous stage to avoid overflowing 45 minute sub-job limit.
+    # Part 2: Split from previous stage to avoid overflowing 45 minute sub-job limit.
```

Comment:
Maybe change `-PROJECT=Firebase` to `-PROJECT=FirebaseInstanceID` at line 196?

Reply:
Or maybe add something a variable to identify parts? Seeing jobs with the same identifiers is a bit confusing to me.